pkg/proxy/iptables/proxier.go

```diff
@@ -1445,36 +1415,8 @@ func (proxier *Proxier) syncProxyRules() {
 			)
 			proxier.natRules.Write(args)
 		} else {
-			// First write session affinity rules only over local endpoints, if applicable.
-			if svcInfo.SessionAffinityType() == v1.ServiceAffinityClientIP {
-				for _, endpointChain := range localEndpointChains {
-					proxier.natRules.Write(
-						"-A", string(svcXlbChain),
-						"-m", "comment", "--comment", svcNameString,
-						"-m", "recent", "--name", string(endpointChain),
-						"--rcheck", "--seconds", strconv.Itoa(svcInfo.StickyMaxAgeSeconds()), "--reap",
-						"-j", string(endpointChain))
-				}
-			}
-
-			// Setup probability filter rules only over local endpoints
-			for i, endpointChain := range localEndpointChains {
-				// Balancing rules in the per-service chain.
-				args = append(args[:0],
-					"-A", string(svcXlbChain),
-					"-m", "comment", "--comment",
-					fmt.Sprintf(`"Balancing rule %d for %s"`, i, svcNameString),
```

Comment:
@danwinship do we care that we're losing a bit of context in the rule comment here?

Reply:
no; that was only for the rules from the XLB chain (ie, external endpoints of `externalTrafficPolicy: Local` services). We didn't have a comment like that in the much more common case of rules from the SVC chain.

(Or maybe we used to have a comment like this in the SVC case, and it got copied to the XLB case, and then later got removed from the SVC case but accidentally not the XLB case as part of the reducing-comment-memory-usage change?)